andela/wg-gnext
# #154239884 Update the given muscles with their exercises

#### What does this PR do?
This PR updates the muscles list

#### Description of Task to be completed?
Add the following muscles as well as their exercises:
- Lateral Deltoid - 17
- Obliquus externus abdominis - 18
- Posterious Deltoid - 19
- Forearms - 20

#### How should this be manually tested?
- Clone this branch
- Inside your virtual environment, run
```
$ invoke load-fixtures -s ./settings.py
```
 - This adds the muscles and their corresponding exercises to your database. 
- Start the application server, login and from the `Exercises` tab, click on `Muscle overview`.
- You should be able to see the above listed muscles.

#### Any background context you want to provide?
The muscle `Obliquus externus abdominis` was already existing in the database. I therefore found  no need of including it in the update.

#### What are the relevant pivotal tracker stories?
[#154239884](https://www.pivotaltracker.com/story/show/154239884)

#### Screenshots (if appropriate)
- Forearms Muscle
<img width="1440" alt="screen shot 2018-02-01 at 3 50 54 pm" src="https://user-images.githubusercontent.com/29917291/35680574-d5717654-076b-11e8-8f02-cf6898cf9ef2.png">

- Lateral deltoid
<img width="1440" alt="screen shot 2018-02-01 at 3 51 06 pm" src="https://user-images.githubusercontent.com/29917291/35680620-ee636898-076b-11e8-8b20-ce674f06cd24.png">

- Posterious Deltoid 
<img width="1440" alt="screen shot 2018-02-01 at 3 51 15 pm" src="https://user-images.githubusercontent.com/29917291/35680640-f7559bba-076b-11e8-9b38-e29359bcb651.png">
